website/_kb/user-guide/dashboards/dashboard-editing.md

```diff
@@ -9,41 +9,33 @@ toc: true
 
 ## Creating a Dashboard
 
-A dashboard is defined as a combination of visualizations and text boxes. Want
-to know more? Keep reading!
+A dashboard lets you combine visualizations and text boxes that provide context with your data. Want to know more? Keep reading!
```

Comment:
Maybe time to retire the "want to know more?" part? :-)

Reply:
Removed in my latest commit.